docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -72,6 +72,15 @@ General-purpose aggregation functions
 
         Required argument. Needs to be a string.
 
+    .. attribute:: distinct
+
+        .. versionadded: 1.10
+
+        This is an optional boolean argument.
```

Comment:
An optional boolean argument that determines if concatenated values will be distinct. Defaults to False.